web/templates/pages/member_page.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+
+{% load staticfiles %}
+
+{% block title %}TrackCat Member{% endblock %}
+
+{% block content%}
+
+	<div class="row">
+		<div class="member_profile">
+			<div class="col-md-12">
+			<h2>{% if user.fullname %}
+			{{ user.fullname }} 
+			{% else %}
+			{{ user.username }}
+			{% endif %}'s profile</h2>
+			{% for field in form %}
+			{{ field.error }}
+			{% endfor %}
+			</div>	
+		</div>
+	</div>
+	<div class="row">
+		<div class="col-md-4 col-xs-6">
+			<img src="../../static/img/{{user.profile_picture}}">
+			<p><strong>GitHub Name:</strong></p>
+			<p>{{ user.githubuser }}</p>
+			<p><strong>Emal:</strong></p>
+			<p>{{ user.email }}</p>
+			<p><strong>Web Site:</strong></p>
+			<p>{{ user.website }}</p>
+		</div>
+		<div class="col-md-8 col-xs-6">
+			<p><strong>Join date:</strong>{{ user.date_created|date:"d.m.Y" }}</p>
+			<p><strong>Member's Status:</strong></p>
+			<p>{{ user.activity_status }}</p>
+			<p><strong>Biography:</strong></p>
+			<p>{{ user.bio }}</p>
```

Comment:
hm, i think so, we defined when talking about models and mockup, that we want to have bio. I think it is a good opportunity that every Cat explains why coding and how is coding changing our life. Maybe we inspire people to join us and expand our mission. Is not this a point of CC, inspire the people and change the world :) maybe we should replace the word with more appropriate one. Will find one... (and I am not arguing :) )